store/postgres/src/subgraph_store.rs

```diff
@@ -923,6 +935,44 @@ impl SubgraphStoreInner {
         store.get_proof_of_indexing(site, indexer, block).await
     }
 
+    pub(crate) async fn get_public_proof_of_indexing(
+        &self,
+        id: &DeploymentHash,
+        block_number: BlockNumber,
+        block_store: Arc<impl BlockStore>,
+    ) -> Result<Option<(PartialBlockPtr, [u8; 32])>, StoreError> {
+        let (store, site) = self.store(&id).unwrap();
```

Comment:
We should probably propagate the error with `?`.

Reply:
Oh, yes.